### PR TITLE
Update eslint, downgrade copywebpack, remove uglify

### PR DESCRIPTION
**Pull request by leemyongpakvn**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Upgrade `eslint` from 6.8.0 to 7.5.0, upgrade related dependencies as well <br> Downgrade `copy-webpack-plugin` to 6.4.1 because 11.0.0 is not compatible with webpack 4.46 <br> Remove deprecated and unused `uglify` <br> Upgrade `babel/core` from 7.15.8 to 7.21.0, upgrade related dependencies and config as well <br> Upgrade `eslint-config-prestashop` to 0.2.1 with correct peer Dependencies
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #{issue URL here}, Fixes #{another issue URL here}
| Sponsor company   | Your company or customer's name goes here (if applicable).
| How to test?      | Apply PR changes; then run `npm install` && `npm run build` in Node 14/15/16.

### CODE REVIEW

**Review by lartist — APPROVED** [review]

File: _dev/front/js/components/Button/Button.vue [hunk 1]
```diff
@@ -169,7 +170,7 @@
             '.product-quantity input#quantity_wanted',
           );
 
-          this.productAttributeId = parseInt(args.id_product_attribute, 10);
+          this.idProductAttribute = parseInt(args.id_product_attribute, 10);
```
Comment (kpodemski) [inline _dev/front/js/components/Button/Button.vue:173]: @leemyongpakvn what was the point of this change?

Comment (leemyongpakvn) [reply _dev/front/js/components/Button/Button.vue:173]: @kpodemski It is a specific rule of Vue named [vue/no-mutating-props](https://eslint.vuejs.org/rules/no-mutating-props.html)
You can see similar thing with `idList` at  line 73 and line 158.

**Review by kpodemski — APPROVED** [review]

**Review by MhiriFaten — APPROVED** [review]
Hello @leemyongpakvn ,

Thank for your PR, LGTM.
It is QA approved :heavy_check_mark:

**Comment by nicosomb** [comment]
Thank you @leemyongpakvn

**Pull request merged**